django/db/models/sql/compiler.py

```diff
@@ -356,7 +356,7 @@ def get_order_by(self):
             # not taken into account so we strip it. When this entire method
             # is refactored into expressions, then we can check each part as we
             # generate it.
-            without_ordering = self.ordering_parts.search(sql).group(1)
+            without_ordering = self.ordering_parts.search("".join(sql.splitlines())).group(1)
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

It seems a code comment may also be useful.